content/docs/1.6.0/volumes-and-nodes/storage-class-parms.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Storage Class Settings
+weight: 3
+---
+
+## Overview
+
+Storage Class as a resource object has a number of settable parameters.  See also [Kubernetes Storage Class](https://kubernetes.io/docs/concepts/storage/storage-classes).  
+Most of the ones in this list are specific to Longhorn's driver.  A few are common to all Kubernetes storage classes:
+
+### Default Class
```

Comment:
If I am not mistaken, whether a StorageClass is default or not is controlled by a label, not a field. We should make this explicit or (and this may actually be my preference) simply not document it since the Kubernetes docs already handle it fine.